Makefile

```diff
@@ -52,20 +52,18 @@ deps: ## Ensures fresh go.mod and go.sum.
 
 .PHONY: generate
 generate: ## Generate all dynamic files.
-generate: generate-pow generate-sol
+generate: pkg/pow/kernelSource.go pkg/contracts/%/%
```

Comment:
this means run a target called `pkg/pow/kernelSource.go` , `pkg/contracts/%/%` 

`%/%` is a wild char to match `pkg/contracts/tellor/tellor.go` and `pkg/contracts/getter/getter.go`